database/seed.js

```diff
@@ -0,0 +1,24 @@
+const db = require('./index.js');
+
+const seed = () => {
+  return db.navbarSave(seedGen());
+}
```

Comment:
What is the purpose of this seed function? It isn't invoked anywhere in the code, and its operation duplicates what is already happening in the database/index.js file.

Reply:
Legacy code, forgot to remove when I changed the way I was invoking things.